src/Sarif/Autogenerated/ArtifactRoles.cs

```diff
@@ -30,6 +30,7 @@ public enum ArtifactRoles
         Extension = 8192,
         Translation = 16384,
         Taxonomy = 32768,
-        Policy = 65536
+        Policy = 65536,
+        ReportingConfigurationOverride = 131072
```

Comment:
Let's use 'referencedOnCommandLine' instead. Let's ask Larry if we can drop 'responseFile'.